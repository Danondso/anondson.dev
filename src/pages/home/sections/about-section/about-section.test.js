import React from 'react';
import renderer from 'react-test-renderer';
import AboutSection from './about-section';

describe('About Section Test Suite', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AboutSection />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
