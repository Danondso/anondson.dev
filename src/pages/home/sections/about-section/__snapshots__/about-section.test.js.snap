// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Section Test Suite renders correctly 1`] = `
<section
  alt="profile description"
  style={
    Object {
      "backgroundColor": "var(--background-peach)",
      "boxShadow": "0 3px 2px rgba(0, 0, 0, 0.15)",
      "padding": "4rem",
    }
  }
>
  <h1
    style={
      Object {
        "color": "var(--text-color)",
      }
    }
  >
    Hello.
  </h1>
  <div
    style={
      Object {
        "color": "var(--text-color)",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "wrap",
        "justifyContent": "space-between",
      }
    }
  >
    <section
      style={
        Object {
          "fontSize": "1.05rem",
          "maxWidth": "24rem",
          "padding": "1rem",
        }
      }
    >
      I'm Dublin Anondson. I'm a developer living in
      <a
        className="link"
        href="https://northwestarkansas.org/"
        rel="noopener noreferrer"
        target="_blank"
      >
         Northwest Arkansas 
      </a>
      with my lovely wife Rachel. It's gorgeous here but sometimes we wish we were closer to some mountains ⛰️. I write code up the street at
      <a
        className="link"
        href="https://movista.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
         Movista. 
      </a>
    </section>
    <section
      style={
        Object {
          "fontSize": "1.05rem",
          "maxWidth": "24rem",
          "padding": "1rem",
        }
      }
    >
      I'm usually juggling a couple personal projects at a time, which you can find on
      <a
        className="link"
        href="https://github.com/Danondso/"
        rel="noopener noreferrer"
        target="_blank"
      >
         GitHub. 
      </a>
      Including a
      <a
        className="link"
        href="https://pedantic-fermat-392953.netlify.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
         movie search app 
      </a>
      and a minimalist
      <a
        className="link"
        href="https://github.com/Danondso/basic-music-player"
        rel="noopener noreferrer"
        target="_blank"
      >
         spotify clone. 
      </a>
      Some of my past professional work includes shipping a user management system for
      <a
        className="link"
        href="https://apps.apple.com/us/app/fsbp-track-it/id1458886205"
        rel="noopener noreferrer"
        target="_blank"
      >
         FSBP TrackIt. 
      </a>
    </section>
    <section
      style={
        Object {
          "fontSize": "1.05rem",
          "maxWidth": "24rem",
          "padding": "1rem",
        }
      }
    >
      When not at the computer I'm usually making/listening/experiencing music or exercising. I volunteer some time every month to help others through
      <a
        className="link"
        href="https://askadev.org/"
        rel="noopener noreferrer"
        target="_blank"
      >
         AskADev. 
      </a>
      I also publish some guides (and other stuff) over on
      <a
        className="link"
        href="https://dev.to/danondso"
        rel="noopener noreferrer"
        target="_blank"
      >
         dev. 
      </a>
    </section>
  </div>
</section>
`;
